controller.rst

```diff
@@ -290,6 +290,28 @@ in your controllers.
 
 For more information about services, see the :doc:`/service_container` article.
 
+Generating Controllers
+----------------------
+
+To save time, you can also tell Symfony to generate a new controller class:
+
+.. code-block:: terminal
+
+    $ php bin/console make:controller BrandNewController
+
+    created: src/Controller/BrandNewController.php
+
+If you want to generate an entire CRUD from a Doctrine :doc:`entity </doctrine>`,
+use:
+
+.. code-block:: terminal
+
+    $ php bin/console make:crud Product
+
+.. versionadded::
+
+    The ``make:crud`` command was added in MakerBundle 1.2.
```

Comment:
In the `versionadded` directive we don't leave a blank line after it (maybe it's not allowed or caused some problems in the past?). Also, we decided to standardize the verb used in these messages as `introduced in ...` instead of `added in...`